doc/api.rst

```diff
@@ -58,8 +59,11 @@ The FlowProject
 .. autoclass:: FlowProject
     :show-inheritance:
     :members:
-    :exclude-members: pre,post
+    :exclude-members: pre,post,operation
 
+.. automethod:: flow.FlowProject.operation(func, name=None)
```

Comment:
Do you have to specify the arguments, or can sphinx detect them automatically?

Reply:
Have to